doc/releases/changelog-dev.md

```diff
@@ -79,6 +79,11 @@
   when at the beginning of a circuit, thus behaving like `StatePrep`.
   [(#4583)](https://github.com/PennyLaneAI/pennylane/pull/4583)
 
+* The new device API can now take a `prng_key` kwarg, to set the key for the JAX pseudo random 
```

Comment:
Are we changing the new device API, or the new default qubit?

Reply:
good point, thanks!